ddl/db_integration_test.go

```diff
@@ -2155,7 +2153,7 @@ func (s *testIntegrationSuite7) TestAddExpressionIndexOnPartition(c *C) {
 	tk := testkit.NewTestKit(c, s.store)
 	tk.MustExec("use test")
 	tk.MustExec("drop table if exists t;")
-	s.tk.MustExec(`create table t(
+	tk.MustExec(`create table t(
```

Comment:
```
[2020-06-18T09:41:04.699Z] FAIL: db_integration_test.go:2152: testIntegrationSuite7.TestAddExpressionIndexOnPartition
[2020-06-18T09:41:04.699Z] 
[2020-06-18T09:41:04.699Z] db_integration_test.go:2156:
[2020-06-18T09:41:04.699Z]     tk.MustExec(`create table t(
[2020-06-18T09:41:04.699Z]     	a int,
[2020-06-18T09:41:04.699Z]     	b varchar(100),
[2020-06-18T09:41:04.699Z]     	c int,
[2020-06-18T09:41:04.699Z]     	PARTITION BY RANGE ( a ) (
[2020-06-18T09:41:04.699Z]     	PARTITION p0 VALUES LESS THAN (6),
[2020-06-18T09:41:04.699Z]     		PARTITION p1 VALUES LESS THAN (11),
[2020-06-18T09:41:04.699Z]     		PARTITION p2 VALUES LESS THAN (16),
[2020-06-18T09:41:04.699Z]     		PARTITION p3 VALUES LESS THAN (21)
[2020-06-18T09:41:04.699Z]     	);`)
[2020-06-18T09:41:04.699Z] /home/jenkins/agent/workspace/tidb_ghpr_check_2/go/src/github.com/pingcap/tidb/util/testkit/testkit.go:207:
[2020-06-18T09:41:04.699Z]     tk.c.Assert(err, check.IsNil, check.Commentf("sql:%s, %v, error stack %v", sql, args, errors.ErrorStack(err)))
[2020-06-18T09:41:04.699Z] ... value *errors.withStack = [parser:1064]You have an error in your SQL syntax; check the manual that corresponds to your TiDB version for the right syntax to use line 5 column 11 near "PARTITION BY RANGE ( a ) (
```
This case failed, we need to update line2159 and use `c int)` instead of `c int,`.

Reply:
Fixed in https://github.com/pingcap/tidb/pull/18003/commits/f658bbf60f74bad59be711f2b7108934e4d06795

PTAL, thanks!